webmvc25-jetty/src/main/webapp/WEB-INF/applicationContext.xml

```diff
@@ -27,7 +27,7 @@
 
   <!-- Configuration for how to send spans to Zipkin -->
   <bean id="sender" class="zipkin2.reporter.beans.URLConnectionSenderFactoryBean">
-    <property name="endpoint" value="${brave.zipkin.baseUrl}/api/v2/spans"/>
```

Comment:
zipkin-reporter will own the endpoint for zipkin. it will eventually allow and prefer zipkin.endpoint also